base/deprecated.jl

```diff
@@ -220,7 +220,7 @@ for f in (:sin, :sinh, :sind, :asin, :asinh, :asind,
         :tan, :tanh, :tand, :atan, :atanh, :atand,
         :sinpi, :cosc, :ceil, :floor, :trunc, :round,
         :log1p, :expm1, :abs, :abs2,
-        :log, :log2, :log10, :exp, :exp2, :exp10, :sinc, :cospi,
+        #=:log,=# :log2, :log10, #=:exp,=# :exp2, :exp10, :sinc, :cospi,
```

Comment:
The `:log` removal snuck in from another commit, and I will delete `:exp` rather comment it as you prefer. Fixed on push. Thanks! :)